dali/test/python/test_RN50_data_pipeline.py

```diff
@@ -52,50 +70,51 @@ def base_define_graph(self, inputs, labels):
         return (output, labels)
 
 class MXNetReaderPipeline(CommonPipeline):
-    def __init__(self, batch_size, num_threads, device_id, num_gpus, data_paths, prefetch, fp16, nhwc):
-        super(MXNetReaderPipeline, self).__init__(batch_size, num_threads, device_id, prefetch, fp16, nhwc)
-        self.input = ops.MXNetReader(path = data_paths[0], index_path=data_paths[1], shard_id = device_id, num_shards = num_gpus)
+    def __init__(self, batch_size, num_threads, device_id, num_gpus, data_paths, prefetch, fp16, nhwc, decoder_type, cache_size, cache_threshold, cache_type, reader_queue_depth):
```

Comment:
Maybe use `*args, **kwargs` instead of adding more args that are just pass through.

Reply:
will do